src/Workspaces/VisualBasic/Portable/Formatting/DefaultOperationProvider.vb

```diff
@@ -55,9 +55,9 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Formatting
             If previousToken.Kind = SyntaxKind.GreaterThanToken AndAlso previousToken.Parent IsNot Nothing AndAlso TypeOf previousToken.Parent Is AttributeListSyntax Then
 
                 ' If the attribute is followed by another attribute then there is no line operation
-                If currentToken.Kind = SyntaxKind.LessThanToken AndAlso currentToken.Parent IsNot Nothing AndAlso TypeOf currentToken.Parent Is AttributeListSyntax Then
-                    Return Nothing
-                End If
+                'If currentToken.Kind = SyntaxKind.LessThanToken AndAlso currentToken.Parent IsNot Nothing AndAlso TypeOf currentToken.Parent Is AttributeListSyntax Then
```

Comment:
I think new line might only be added when attribute is associated with a statement. not expression.